### PR TITLE
chore(ci): prettier & always include latest built version

### PR DESCRIPTION
**Pull request by pauldambra**
a small PR so we can check we can deploy a change that builds the same code as the last version safely

building up confidence for working towards https://github.com/PostHog/user-agent-plugin/pull/4

### CODE REVIEW

**Comment by marandaneto** [comment]
What does "code-neutral" mean here? a bit lost with the problem definition so I cannot review if the changes make sense.

**Comment by pauldambra** [comment]
it means no functional changes :)

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Weyert de Boer
```
Comment (marandaneto) [inline LICENSE]: I dont think this is allowed, you'd need to keep the original one
https://www.tldrlegal.com/license/mit-license

File: src/__tests__/v2.behavior.test.ts [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
+    Meta,
     PluginEvent,
-    PluginMeta,
     StorageExtension,
     UtilsExtension,
 } from '@posthog/plugin-scaffold'
-import { processEvent } from '../plugin'
+import { processEvent, UserAgentMetaInput } from '../plugin'
+
+function makeMeta(options?: {
```
Comment (marandaneto) [inline src/__tests__/v2.behavior.test.ts:11]: Nice refactoring to avoid code dupe.

**Review by marandaneto — APPROVED** [review]

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+Copyright (c) 2024 PostHog
```
Comment (pauldambra) [inline LICENSE]:
```suggestion
Copyright (c) 2020 Weyert de Boer
```

**Review by xvello — APPROVED** [review]

**Pull request merged**